Documentation/contributing/development/dev_setup.rst

```diff
@@ -82,6 +82,51 @@ Finally, in order to run Cilium locally on VMs, you need:
 | `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_  | >= 5.2                | N/A (OS-specific)                                                              |
 +------------------------------------------------------------+-----------------------+--------------------------------------------------------------------------------+
 
+Kind-based Setup (preferred)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The setup for a `kind <https://kind.sigs.k8s.io/>`_ environment lives in
+``contrib/scripts/kind.sh``. This setup does not require any VMs and/or
+VirtualBox on Linux, but `Docker for Mac
+<https://docs.docker.com/desktop/install/mac-install/>`_ is required for Mac
+OS.
+
+There are Makefile targets that make it easy to use to spin up an environment
+and build Cilium images.
+
+* ``make kind``: Creates a kind cluster based on configuration passed in (see
+  below for configuration).
```

Comment:
s/below/following/g, consistent format: "for more information about `x`, see `y`"
```suggestion
* ``make kind``: Creates a kind cluster based on the configuration passed in. For more information, see _`Configuration for clusters`.
```